From a1924d7fb2a84d35dca8fe64115f84472aef679a Mon Sep 17 00:00:00 2001
From: Felix Radensky <felix.r@variscite.com>
Date: Wed, 11 Dec 2019 12:51:14 +0100
Subject: [PATCH 4/4] imx-atf: Fix i.MX8M/i.MX8MM DRAM timing structure to
 match u-boot

The DRAM timing structure was modified in u-boot. Since u-boot
copies it for later use by ATF, the ATF definition should match
the u-boot one. This fixes broken suspend/resume sequence.
---
 plat/imx/common/include/dram.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/plat/imx/common/include/dram.h b/plat/imx/common/include/dram.h
index bf0b38489..8b5f90c11 100644
--- a/plat/imx/common/include/dram.h
+++ b/plat/imx/common/include/dram.h
@@ -41,6 +41,8 @@ struct dram_cfg_param {
 };
 
 struct dram_timing_info {
+	/* ddrphy firmware offset */
+	unsigned int ddrphy_fw_offset;
 	/* umctl2 config */
 	struct dram_cfg_param *ddrc_cfg;
 	unsigned int ddrc_cfg_num;
-- 
2.17.1

